network/ip_tracker.go

```diff
@@ -446,7 +446,7 @@ func (i *ipTracker) resetBloom() error {
 		return err
 	}
 
-	count := max(maxIPEntriesPerValidator*i.gossipableIDs.Len(), minCountEstimate)
+	count := max(maxIPEntriesPerNode*i.trackedIDs.Len(), minCountEstimate)
```

Comment:
This is a minor bug fix. Previously we weren't accounting for `manuallyTracked` nodes correctly in the bloom filter size. This caused frequent bloom filter resets once we were tracking manually tracking significantly more nodes than we were previously.

See added test.